packages/block-library/src/post-content/edit.js

```diff
@@ -1,9 +1,22 @@
-export default function PostContentEdit() {
+/**
+ * WordPress dependencies
+ */
+import { EntityProvider } from '@wordpress/core-data';
```

Comment:
We should make the entity hooks take an `id` parameter and only fallback to the provided value.

Adding a provider here shouldn't be necessary.

Reply:
Ah, that makes sense. Done 👍